src/server/scripts/Commands/cs_modify.cpp

```diff
@@ -997,6 +998,66 @@ class modify_commandscript : public CommandScript
         target->GiveXP(xp, nullptr);
         return true;
     }
+    	// Edit Player Power 
+	static bool HandleModifyPowerCommand(ChatHandler* handler, const char* args)
+	{
+		int32 power, maxpower;
+		Player* target = handler->getSelectedPlayerOrSelf();
+		int8 const powerMultiplier = 10;
+		if (CheckModifyResources(handler, args, target, power, maxpower, powerMultiplier))
+		{
+			switch (target->getPowerType())
```

Comment:
```
src/server/scripts/Commands/cs_modify.cpp:1009:12:
 fatal error:
 13 enumeration values not handled in switch: 'POWER_COMBO_POINTS', 'POWER_RUNES', 'POWER_SOUL_SHARDS'... [-Wswitch]
                        switch (target->getPowerType())
                                ^
1 error generated.
```
https://github.com/TrinityCore/TrinityCore/blob/2d75d8071bb6d85987bdc5d88ed2271822fcbcfd/src/server/game/Miscellaneous/SharedDefines.h#L250-L274

There are 22 `Powers` listed in the enum in `SharedDefines.h` which means you need to find a way to tell this script to skip/ignore the 13 powers not handled by this switch. I don't know what that code should look like, but maybe some of the other developers will know.

Reply:
Need add a default in switch and add tabs in  cases

Reply:
OK, I get what you mean that it needs a
```cpp
			default:
				break;
```
at the end, but not sure what you mean by `add tabs in cases`

Reply:
Add tabulations in  all cases:
```diff
-    case X
+        case X
```

Sorry mobile :smile:

Reply:
OK, I see. Add correct indent to each case,

Reply:
BTW; changes have been made, but they are all below this comment section
(and hence they don't get hidden by the update).